plugins/sayings.rb

```diff
@@ -13,6 +13,7 @@ def self.messages
       /^(hi|hello|sup|what's up|hiya|hey|wb|welcome back|it's|right|how you doin|ok|<3).*porygon.*$/i => "Porygon! ^_^",
       /^rolling around at the speed of sound$/i => "Got places to go, gotta follow my rainbow",
       /^moan$/i => "GROAN"
+      /^enter.*key$/i => "Shut Up"
```

Comment:
While this would work, keep in mind this regex will match any line that begins with "enter" and ends in "key".

http://rubular.com/r/WM6cKm9TrJ